examples/browser/extension/package.json

```diff
@@ -20,7 +20,5 @@
     "webpack-dev-server": "3.1.14",
     "write-file-webpack-plugin": "4.5.0"
   },
-  "dependencies": {
-    "@aeternity/aepp-sdk": "7.2.1"
-  }
+  "dependencies": {}
```

Comment:
Hm.. isn't this needed?

Reply:
Nope, but it was generated automatically by `npm uninstall`